### PR TITLE
chore(release): release 0.1.374

### PR DESCRIPTION
**Pull request by machi1990**
https://github.com/openshift-online/ocm-sdk-go/pull/851 has to be merged first

### CODE REVIEW

**Comment by machi1990** [comment]
Once this is merged, I'll appreciate a tag to be created as I do not have permission for that.

**Comment by tzvatot** [comment]
Missing update in version.go

**Comment by machi1990** [comment]
Good catch. This has been fixed.

File: CHANGES.md [hunk 1]
```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.374
+- Update model version v0.0.319
```
Comment (tzvatot) [inline CHANGES.md:7]: @machi1990 it's 0.0.326

**Review by tzvatot — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (machi1990) [reply CHANGES.md:7]: Good catch.  I just realised now that I pushed an old branch.

**Review by tzvatot — APPROVED** [review]
LGTM

**Comment by machi1990** [comment]
Thanks @tzvatot

**Pull request merged**